themes/gatsby-theme-blog/src/theme/styles.js

```diff
@@ -0,0 +1,73 @@
+export default {
+  root: {
+    fontFamily: `body`,
+  },
+  Container: {
+    maxWidth: 672,
+    px: 3,
+  },
+  pre: {
+    // references styles from theme.prism (src/theme/prism.js)
+    variant: `prism`,
```

Comment:
Is this kind of like using a parent selector for all the Prism styles? Something like `pre .tag`?

Reply:
Yes, `theme.prism` object is spread into `theme.styles.pre`, which will add child selectors to the `<pre>` tag in MDX.

The `variant` key in `@styled-system/css` is more intended as a way to define styles for something like different button styles, but here it's working more like the `extend` feature from Sass, which I think is unnecessary. Instead this object should probably be imported into the module and spread into this object.

Reply:
yeah, I think especially since these are object styles, a direct import makes more sense.